doc/man/8/ceph.rst

```diff
@@ -506,11 +506,11 @@ Usage::
 
 	ceph osd blocked-by
 
-Subcommand ``create`` creates new osd (with optional UUID).
+Subcommand ``create`` creates new osd (with optional UUID and ID).
```

Comment:
@trociny do you mind adding some warning about the "leave gaps in the id space" problem here ?